README.md

```diff
@@ -93,8 +87,8 @@ To upgrade to the latest version of Flax, you can use:
 
 We provide three examples using the Flax API: a simple multi-layer perceptron, a CNN and an auto-encoder. 
 
-To learn more about the `Module` abstraction, please check our [docs](https://flax.readthedocs.io/), our [broad intro to the Module abstraction](https://github.com/google/flax/blob/master/docs/notebooks/linen_intro.ipynb) or visit our
-[HOWTO guides](https://flax.readthedocs.io/en/latest/howtos.html) page for additional concrete demonstrations of best practices.
+To learn more about the `Module` abstraction, see our [docs](https://flax.readthedocs.io/), our [broad intro to the Module abstraction](https://github.com/google/flax/blob/master/docs/notebooks/linen_intro.ipynb). For additional concrete demonstrations of best practices, see our
+[HOWTO guides](https://flax.readthedocs.io/en/latest/howtos.html).
```

Comment:
Funny enough I don't think we link to this page anywhere from our RTD, so this is the only way the user can access it.

Reply:
I know! I went back and forth about this many times and I can't find a better solution :) Once you're in RTD there's no real need for this link, you just see it in the left navbar. But if you have any suggestions for improvement, happy to hear!